js/src/functions.js

```diff
@@ -2092,6 +2093,34 @@ $(function () {
             $('#copyStatus').remove();
         }, 2000);
     });
+
+    $(document).on('mouseover', '.ajax_notification a', function (event) {
+        Functions.tooltip(
+            $(".ajax_notification"),
+            'span',
+            Messages.strEditQuery
+        );
+    });
+
+    $(document).on('mouseover', '.ajax_notification a.copyQueryBtn', function (event) {
```

Comment:
```suggestion
    $(document).on('mouseover', '.ajax_notification a.copyQueryBtn', function () {
```
You can remove the parameter for all functions that do not use it

Reply:
Done 👌🏻